### DISCUSSION TITLE
How to install a package by specifying a constraint file through the Poetry?

### DISCUSSION THREAD

**Discussion by LGDHuaOPER** [python-poetry/poetry#7615] [Q&A] [answered]
For example: [apache-airflow](https://pypi.org/project/apache-airflow/#installing-from-pypi)

It can be installed specifying a constraint file by using `--constraint` :

```bash
pip install "apache-airflow[celery]==2.5.1" --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-2.5.1/constraints-3.7.txt"
```

Whether Poetry can implement similar functions?

Thanks!

**Comment by radoering** [comment, marked as answer]
Searching the issues for apache-airflow you will find #7051

**Reply by LGDHuaOPER** [reply]
Thanks!
